radix/node.go

```diff
@@ -129,14 +129,21 @@ func (n Node) Insert(path string, key uint64) Node {
 			return n
 		}
 
-		n.children = append([]Node{
-			{
+		if len(n.children) > 0 {
+			n.children = append(n.children, Node{
```

Comment:
hm, for it all if-else construct equivalent to:

```
n.children = append(n.children, Node{
	path: path[:start],
	children: []Node{
		updateParamNode(Node{kind: param}, path[start:], key),
	},
}
```

Reply:
@SuddenGunter dont forget about this one ^^

Reply:
fixed